src/Components/Benefits.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import styled from 'styled-components'
+import Back from '../images/jade-stephens-N21356amsyw-unsplash.jpg'
+import Line from '../images/Line.svg'
+
+const BenefitsWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+  @media (min-width: 768px) {
+    flex-direction: row;
+    padding: 2rem;
+  }
+`
+
+const Text = styled.div`
+  background-image: url("../images/Blob.png");
```

Comment:
It's generally better practice to import images as modules, rather than using their file path directly. This allows webpack to optimize the images and handle them more efficiently.